test/http2/test_frames.py

```diff
@@ -1,50 +1,85 @@
-from netlib.h2.frame import *
-import tutils
-
+from test import tutils
 from nose.tools import assert_equal
+from netlib.http2.frame import *
+
+
+class FileAdapter(object):
```

Comment:
Minor nitpick: The name "FileAdapter" doesn't really convey what it does. How about this?
```python
def hex_to_file(data):
    data = data.decode("hex")
    return tcp.Reader(cStringIO.StringIO(data))
```